Specs/DateTime/Spanish/TimeExtractor.json

```diff
@@ -520,21 +520,21 @@
     "Input": "Volvere a la madrugada",
     "Results": [
       {
-        "Text": "a la madrugada",
+        "Text": "madrugada",
```

Comment:
"Madrugada" is the period of early morning. Should also be moved along with the others.

Reply:
"Dawn", would be something like "el amanecer".
"amanecer by itself is a verb though.

Reply:
I see, replaced with "media madrugada" (it seems to be used in Spanish) and added support for MidEarlyMorning in TimeParser.